packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator/src/Providers/MethodProvider.cs

```diff
@@ -54,10 +56,23 @@ public MethodProvider(MethodSignature signature, ValueExpression bodyExpression,
         {
             Signature = signature;
             BodyExpression = bodyExpression;
-            XmlDocs = xmlDocProvider ?? MethodProviderHelpers.BuildXmlDocs(signature.Parameters, signature.Description, signature.ReturnDescription, null);
+            _xmlDocs = xmlDocProvider;
             EnclosingType = enclosingType;
         }
 
+        private XmlDocProvider? BuildXmlDocs()
+        {
+            return MethodProviderHelpers.BuildXmlDocs(
+                Signature.Parameters,
+                Signature.Description,
+                Signature.ReturnDescription,
+                BodyStatements != null ?
+                    MethodProviderHelpers.GetParamhash(
```

Comment:
nit - is it possible to update this helper ?
```suggestion
                    MethodProviderHelpers.GetParamHash(
```